pkg/artifacts/registry.go

```diff
@@ -2,42 +2,57 @@ package artifacts
 
 import (
 	"context"
-	"fmt"
 
-	"oras.land/oras-go/pkg/content"
-	"oras.land/oras-go/pkg/oras"
+	"github.com/go-logr/logr"
+
+	"github.com/aws/eks-anywhere-packages/pkg/registry"
 )
 
+const certFile = "/tmp/registry-mirror/CACERTCONTENT"
```

Comment:
We need to read this from a secret config, right?

Reply:
The secret is mounted on /tmp/registry-mirror as a file

Reply:
It has been updated to /tmp/config/registry/ca.crt

Reply:
I don't think it is a good idea to use a copy when we can use the original. This version is a lot more likely to be correct.